Source/Lib/Codec/EbEncHandle.c

```diff
@@ -3312,6 +3314,13 @@ static EB_ERRORTYPE CopyFrameBuffer(
         EB_U16                           sourceCbStride   = (EB_U16)(inputPtr->cbStride);
 
         //EB_U16                           lumaHeight  = inputPicturePtr->maxHeight;
+
+        if (lumaWidth > lumaStride || lumaWidth > sourceLumaStride || chromaWidth > chromaStride)
+        {
```

Comment:
Hi Mark, we are using
if() {
}
Since it is not easily correct by the style check script, could you fix them in this patch?